### ISSUE TITLE
How to use

### ISSUE THREAD

**Issue by silver2791** [KurtSanders/STBalboaSpaControl#1]
Hi Kurt,

Just wondering what is the best way to use this code?

Thanks

**Comment by KurtSanders** [comment]
Dear silver2791,

I updated the [ReadMe file](https://github.com/KurtSanders/STBalboaSpaControl) to help with this initial "view only" version.  I plan to enhance over the next few weeks for state operation control.  

Is your spa enabled with a compatible Balboa™ WiFi controller as I have indicated in the ReadMe?

**Comment by silver2791** [comment]
Dear Kurt,

This is awesome.  Thanks.

Are you looking at a version which the spa temp can be change the desired spa temp to a higher or lower temp?

Thanks again
S

**Comment by KurtSanders** [comment]
Yes.  Just a matter of devoting some time to add this capabilities!  Does this version display all the values for your Spa?

**Comment by KurtSanders** [comment]
Just uploaded a new version 1.01 that controls spa light on/off and Heating SetPoint Up/Down.  Still working on turning Pump1 and Pump2 on/off.  What model spa to you have?  Does it have 2 Pumps like my Bullfrog Spa?
